doc/jsk_2015_05_baxter_apc/apc2015_demo_sim.rst

```diff
@@ -32,7 +33,7 @@ Demo
 .. code-block:: bash
 
   roslaunch jsk_2015_05_baxter_apc baxter_sim.launch
-  roslaunch jsk_2015_05_baxter_apc setup.launch
+  roslaunch jsk_2015_05_baxter_apc setup_head.launch
```

Comment:
MEMO: `setup.launch` was renamed to `setup_head.launch` at https://github.com/start-jsk/jsk_apc/commit/12259b53b358d2a9b39e57e18ebbc2b02fa5bb51